src/guiKeyChangeMenu.cpp

```diff
@@ -105,15 +104,12 @@ void GUIKeyChangeMenu::removeChildren()
 {
 	const core::list<gui::IGUIElement*> &children = getChildren();
 	core::list<gui::IGUIElement*> children_copy;
-	for (core::list<gui::IGUIElement*>::ConstIterator i = children.begin(); i
-		 != children.end(); i++)
-	{
-		children_copy.push_back(*i);
+	for (gui::IGUIElement*i : children) {
```

Comment:
copy not possible: "error: no type named 'value_type' in 'irr::core::list<irr::gui::IGUIElement *>'"